hack/test-gce/nodeagent_install.sh

```diff
@@ -0,0 +1,249 @@
+    sudo mkdir -p /etc/cni/net.d
+    sudo touch /etc/cni/net.d/10-containerd-net.conflist
+    sudo chmod a+x /etc/cni/net.d/10-containerd-net.conflist
+cat << EOF | sudo tee /etc/cni/net.d/10-containerd-net.conflist
+{
+  "cniVersion": "0.4.0",
+    "name": "containerd-net",
+    "plugins": [
+    {
+      "type": "bridge",
+      "bridge": "cni0",
+      "isGateway": true,
+      "ipMasq": true,
+      "promiscMode": true,
+      "ipam": {
+        "type": "host-local",
+        "ranges": [
+          [{
+            "subnet": "10.22.0.0/16"
+          }]
+        ],
+        "routes": [
+        { "dst": "0.0.0.0/0" }
+        ]
+      }
+    },
+    {
+      "type": "portmap",
+      "capabilities": {"portMappings": true}
+    }
+  ]
+}
+EOF
+
+    sudo systemctl restart containerd    
+}
+
+runsc_install() {
+	echo -e "## Download and Install runsc (gVisor).\n"
+	(
+	  set -e
+	  ARCH=$(uname -m)
+	  URL=https://storage.googleapis.com/gvisor/releases/release/latest/${ARCH}
+	  wget ${URL}/runsc ${URL}/runsc.sha512 \
+		${URL}/containerd-shim-runsc-v1 ${URL}/containerd-shim-runsc-v1.sha512
+	  sha512sum -c runsc.sha512 \
+		-c containerd-shim-runsc-v1.sha512
+	  rm -f *.sha512
+	  chmod a+rx runsc containerd-shim-runsc-v1
+	  sudo mv runsc containerd-shim-runsc-v1 /usr/local/bin
+	)
+	# To install gVisor as a Containerd runtime, run the following commands:
+	# /usr/local/bin/runsc install
+	sudo systemctl restart containerd
+}
+
+kata_install(){
+	ARCH=$(arch)
+	BRANCH="${BRANCH:-master}"
+	sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/xUbuntu_$(lsb_release -rs)/ /' > /etc/apt/sources.list.d/kata-containers.list"
+	curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+	sudo -E apt-get update
+	sudo -E apt-get -y install kata-runtime kata-proxy kata-shim
+}
+
+
+golang_tools(){
+   if [ "$(go version)" != "go version go1.18.7 linux/amd64" ] > /dev/null 2>&1
+    then
+       echo -e "## INSTALLING GOLANG TOOLS FOR FORNAXCORE AND NODEAGENT"
+       sudo apt -y install make gcc jq > /dev/null 2>&1
+	   echo "Install golang."
+       wget https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz -P /tmp
+       sudo tar -C /usr/local -xzf /tmp/go${GO_VERSION}.linux-amd64.tar.gz
+       # echo -e 'export PATH=$PATH:/usr/local/go/bin\nexport GOPATH=/usr/local/go/bin\nexport KUBECONFIG=/etc/kubernetes/admin.conf' |cat >> ~/.bashrc
+	   mkdir ~/go
+	   echo -e '\n' >> ~/.bashrc
+	   echo export GOROOT=\"/usr/local/go\" >> ~/.bashrc
+	   echo export GOPATH=\"\$HOME/go\" >> ~/.bashrc
+	   echo export GOBIN=\"\$HOME/go/bin\" >> ~/.bashrc
+	   echo export PATH=\"/usr/local/go/bin:\$HOME/go/bin:\$PATH\" >> ~/.bashrc
+       source $HOME/.bashrc
+       echo -e "## DONE\n"
+    else
+       echo -e "## go${GO_VERSION} already installed\n "
+   fi
+}
```

Comment:
we do not need to install development tools, you should just scp local image into VM